speech_to_text/speech_to_text.livemd

```diff
@@ -0,0 +1,217 @@
+# Speech to text
+
+```elixir
+Logger.configure(level: :info)
+
+Mix.install(
+  [
+    :bumblebee,
+    :exla,
+    :membrane_core,
+    :membrane_portaudio_plugin
+  ],
+  config: [
+    nx: [default_backend: EXLA.Backend]
+  ]
+)
+```
+
+## Prerequisites
+
+On Linux, you need to install PortAudio, for example with `sudo apt install portaudio19-dev`.
+
+## Introduction
+
+This livebook example shows how to perform a real-time speech-to-text conversion with the use of the [Membrane Framework](https://github.com/membraneframework) and the [Bumblebee](https://github.com/elixir-nx/bumblebee).
+
+You will see how to fetch the audio from your microphone, perform preprocessing and create your own Membrane element that runs an AI speech-to-text conversion model.
+
+## Element performing a speech to text
+
+We need to write a custom sink that will perform speech-to-text and display the resulting transcription. Let's call this element `SpeechToText`.
```

Comment:
Let's link for instance there: https://membrane.stream/learn/glossary#sink
```suggestion
We need to write a custom [sink](https://membrane.stream/learn/glossary#sink) that will perform speech-to-text and display the resulting transcription. Let's call this element `SpeechToText`.
```

Reply:
I have just realized that the `SpeechToText` is not a sink, but a filter :D (I believe there are some other places where we refer to the `SpeechToText` element as "sink")